### DISCUSSION TITLE
How to use jetson-voice on Jetbot

### DISCUSSION THREAD

**Discussion by Myrgued** [NVIDIA-AI-IOT/jetbot#573] [Q&A]
Hi there! I'd like to use the jetson-voice library, that comes as a docker container, on a jetbot within jupyter notebook. Since I am quite new to Jetson and Docker and Python phew, I have no clue how to do that. So far I have a jetbot up and running and went through all the tutorials. I also managed to get the jetson-voice container up and running. But I don't know how to connect it to the Jetbot environment to combine the speech commands with robot actions. Happy, if anyone can help :).

**Comment by dusty-nv** [comment]
@Myrgued I would probably try rebasing the jetbot container against jetson-voice, so then you can use jetson-voice from the jetbot container.

**Reply by Myrgued** [reply]
@dusty-nv Thanks for your responce! It worked for an older version of Jetbot, though its seems some dependancy issues left. I will try to look into that as soon as I have more spare time :).
